Testscripts/Windows/VERIFY-DPDK-VPP.ps1

```diff
@@ -0,0 +1,121 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
```

Comment:
It is nice to put some brief information about VPP. After reviewing code lines, it is enough to understand. The extra description can expedite the reviewer's comprehension speed.

Reply:
I ve put the info